src/styles/blocks/body.scss

```diff
@@ -0,0 +1,10 @@
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  color: #{$color-accent};
+  margin: 0;
+  padding: 0;
```

Comment:
the body doesn't have padding
also, you already reset the margin in the reset file
```suggestion
```